src/common/types/session.type.ts

```
@@ -1,21 +1,18 @@
-    videoUrl: string | null;
-    fileUrl: string | null;
+    videoUrl: string;
+    fileUrl: string;
     likeCount: number;
     viewCount: number;
-
-    user: User;
-}
+    createdAt: Date;
+    updatedAt: Date;
+} 
```

ESLint

[error] Replace `·` with `⏎`

(prettier/prettier)